src/main/java/com/github/algobot76/surabaya/util/StaticAnalysisVisitor.java

```diff
@@ -108,6 +125,34 @@ public void visit(ConstructorDeclaration n, Project project) {
 
 	}
 
+	private String getSrcFromRange(Range range) {
+		System.out.println("getSrcFromRange :");
+		System.out.println(range);
+		StringBuilder sb = new StringBuilder();
+
+		// JavaParser Positions are 1-indexed
+		// https://www.javadoc.io/doc/com.github.javaparser/javaparser-core/3.2.7/com/github/javaparser/Position.html
+
+		String firstLineStr = this.currentFileSrc[range.begin.line - 1];
+		sb.append(firstLineStr.substring(range.begin.column - 1));
+
+		int currLine = range.begin.line;
+		int lastLine = range.end.line - 1;
+		while (currLine < lastLine) {
+			sb.append(System.lineSeparator());
+			sb.append(this.currentFileSrc[currLine]);
+			++currLine;
+		}
+		String lastLineStr = this.currentFileSrc[lastLine];
+
+		// https://stackoverflow.com/questions/52559564/call-to-substring-is-redundant
+		// This does the same as lastLineStr.substring(0, range.end.column);
+		sb.append(lastLineStr, 0, range.end.column);
+		sb.append(System.lineSeparator());
+
+		return sb.toString();
+	}
+
```

Comment:
We could simply use `toString` method to get the source code of a method/constructor. This helper method seems unnecessary.